lang/wasm_alien_thread_runner.cc

```diff
@@ -11,12 +11,15 @@
 #include <seastar/core/posix.hh>
 #include <seastar/core/reactor.hh>
 
+#include "log.hh"
```

Comment:
might want to `#include <unistd.h>` , we should rely on the included headers to do this for us. they could remove their `#include <unistd.h>` at anytime without noticing us beforehand.

Reply:
Ok